tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,46 @@
+def display_inventory(inventory):
+    print("Inventory:")
+    item_total = 0
+    for item_name, item_count in inventory.items():
+        print(item_count, item_name)
+        item_total += item_count
+    print("Total number of items: " + str(item_total))
+    if item_total >= 80:
+        print("CAUTION: You are overloaded, can't move!")
+    elif item_total >= 70:
+        print("CAUTION: Your equipment is very heavy, you're moving slower than usual!")
+    elif item_total >= 60:
+        print("CAUTION: Your backpack weighs a lot, your stamina runs out quicker!")
+
+
+def add_to_inventory(inventory, added_items):
+    skipped = {}
+    added_count = 0
+    for item_name in added_items:
+        if item_name in ['chewed gum', 'rubbish', 'used tissue']:
```

Comment:
I think this should be a CONST, also we can define it at the very top of the file as it doesn't change between function's runs.

You might also want to rework this function, so it takes 3 arguments: inventory, added_items, and unwanted_items.

Up to you, but in current case I'd move it to the const out of the function's scope